kubernetes_state/datadog_checks/kubernetes_state/kubernetes_state.py

```diff
@@ -155,6 +171,11 @@ def check(self, instance):
             self.monotonic_count(scraper_config['namespace'] + '.job.succeeded', job.count, list(job_tags))
             job.set_previous_and_reset_current_ts()
 
+        for job_tags, job in iteritems(self.cron_job_complete_check):
+            status = self.UNKNOWN if job.status is None else job.status
+            self.service_check(scraper_config['namespace'] + '.cronjob.complete', status, list(job_tags))
```

Comment:
The service check name can be confusing, maybe `cronjob.last_job_complete` ?

Reply:
Sure. Will change.